### PR TITLE
Fix upgrade URL for 3.2 RC2

### PR DESCRIPTION
**Pull request by dLobatog**
The upgrade URL pointed to a 404

### CODE REVIEW

**Comment by ehelms** [comment]
:+1:

**Comment by dLobatog** [comment]
Just updated the Foreman packages URL too, it was broken as well and added a small disclaimer about el6.

**Comment by dLobatog** [comment]
Hold on a sec before merging - I saw a few other broken links (capsule upgrade)

**Comment by dLobatog** [comment]
Alrighty, now it'd be ready

**Comment by dLobatog** [comment]
@Katello/website :wave:

**Comment by ehelms** [comment]
ACK - @johnpmitsch ?

File: docs/upgrade/clients.md [hunk 1]
```diff
@@ -21,38 +21,31 @@ Update the Katello client release packages:
...
 
 <div id="el7" style="display:none;" markdown="1">
 ```bash
-yum update -y http://fedorapeople.org/groups/katello/releases/yum/{{ site.version }}/client/RHEL/7Server/x86_64/katello-client-repos-latest.rpm
+yum update -y http://fedorapeople.org/groups/katello/releases/yum/{{ site.version }}/client/el7/x86_64/katello-client-repos-latest.rpm
 ```
 </div>
 
-<div id="fc20" style="display:none;" markdown="1">
+<div id="fc23" style="display:none;" markdown="1">
```
Comment (johnpmitsch) [inline docs/upgrade/clients.md]: @ehelms do we want to add fedora 24? Don't we support that as a client?

Comment (ehelms) [inline docs/upgrade/clients.md]: We do -- but not for upgrades. So we should add it to the installation docs.

Comment (johnpmitsch) [inline docs/upgrade/clients.md]: ah good point

**Comment by johnpmitsch** [comment]
ACK

**Comment by dLobatog** [comment]
ping?

**Pull request merged**